spec/factory_bot/aliases_spec.rb

```diff
@@ -1,29 +1,47 @@
 describe FactoryBot, "aliases" do
-  context "aliases for an attribute" do
-    subject { FactoryBot.aliases_for(:test) }
-    it      { should include(:test) }
-    it      { should include(:test_id) }
+  it "aliases for an attribute should include the original attribute" do
+    subject = FactoryBot.aliases_for(:test)
+
+    expect(subject).to include(:test)
+  end
+
+  it "aliases for an attribute should include the original attribute suffixed with '_id'" do
+    subject = FactoryBot.aliases_for(:test)
+
+    expect(subject).to include(:test_id)
+  end
+
+  it "aliases for a foreign key should include the un-suffixed attribute name" do
+    subject = FactoryBot.aliases_for(:test_id)
+
+    expect(subject).to include(:test)
   end
 
-  context "aliases for a foreign key" do
-    subject { FactoryBot.aliases_for(:test_id) }
-    it      { should include(:test) }
-    it      { should include(:test_id) }
+  it "aliases for a foreign key should include the original attribute name" do
+    subject = FactoryBot.aliases_for(:test_id)
+
+    expect(subject).to include(:test_id)
   end
 
-  context "aliases for an attribute starting with an underscore" do
-    subject { FactoryBot.aliases_for(:_id) }
-    it      { should_not include(:id) }
+  it "aliases for an attribute starting with an underscore should not include a non-underscored version" do
+    subject = FactoryBot.aliases_for(:_id)
+
+    expect(subject).not_to include(:id)
   end
 end
 
 describe FactoryBot, "after defining an alias" do
-  before do
+  it "the list of aliases should include an un-suffixed equivalent" do
     FactoryBot.aliases << [/(.*)_suffix/, '\1']
+    subject = FactoryBot.aliases_for(:test_suffix)
+
+    expect(subject).to include(:test)
   end
 
-  subject { FactoryBot.aliases_for(:test_suffix) }
+  it "the list of aliases should include the original name plus an '_id' suffix" do
```

Comment:
This description is very similar to the one on line  8. Is this testing something else? I'm a little confused by this test.